unitTests/utils/AccessibleButtonTest.ts

```diff
@@ -1,8 +1,19 @@
 import { AccessibleButton } from '../../src/utils/AccessibleButton';
-import { Dom, $$, KEYBOARD, Component } from '../../src/Core';
+import { Dom, $$, KEYBOARD, Component, Defer } from '../../src/Core';
 import { Simulate } from '../Simulate';
 import { ComponentEvents } from '../../src/ui/Base/Component';
 
+function dispatchFakeKeyEvent(element: HTMLElement, eventName: 'keydown' | 'keyup', props: any) {
```

Comment:
Isn't that what the Simulate.keyEvent is doing?

Reply:
Yes, but this one also allows me to define a separate `target` and a custom `preventDefault`, which is important for those tests.